pages/cpm.vue

```diff
@@ -77,6 +83,16 @@ export default {
       storyboards: []
     }
   },
+  computed: {
+    crumbs () {
+      // assembly full path of current page, including the tag
+      const fullPath = ('/data-catalogue' + this.$route.fullPath)
+      // break full path into crumbs for breadcrumbs view
+      const crumbs = fullPath.substring(1).split('/')
```

Comment:
Here it would be nice if the selected panel is also visible on the breadcrumb trail. Even better: it would be absolutely great if the tab was also reflected in the URL. (that might also make it easier to move the breadcrumbs to the header).

Reply:
Good idea! Now the panel is visible and the tab is also reflected in the URL.